ctru-rs/src/error.rs

```diff
@@ -6,8 +10,29 @@ use std::ops::{ControlFlow, FromResidual, Try};
 
 use ctru_sys::result::{R_DESCRIPTION, R_LEVEL, R_MODULE, R_SUMMARY};
 
+/// Custom type alias for generic [`ctru-rs`](crate) operations.
+///
+/// This type is compatible with `ctru-sys::Result` codes.
```

Comment:
Can we do intra-doc linking with `` [`ctru_sys::Result`] `` for references like this?

Reply:
They do locally but I was unsure whether it would work in the `docs.rs` environment (though that may be a problem for another time...).